devicetypes/sky-nie/min-smart-plug.src/min-smart-plug.groovy

```diff
@@ -120,23 +119,11 @@ def updated() {
 
         logDebug "updated()..."
 
-        initialize()
-
         runIn(5, executeConfigureCmds, [overwrite: true])
     }
     return []
 }
 
-private initialize() {
-    def checkInterval = ((60 * 60 * 3) + (5 * 60))
-
-    def checkIntervalEvt = [name: "checkInterval", value: checkInterval, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"]]
-
-    if (!device.currentValue("checkInterval")) {
-        sendEvent(checkIntervalEvt)
-    }
-}
-
```

Comment:
I didn't want you to remove whole `initialize()` method (and what's more important, the `checkInterval` event). I've just ask you to remove only `offlinePingable: "1"` option.

Reply:
Sorry, I misunderstood, I modified it